packages/neon_framework/lib/src/utils/request_manager.dart

```diff
@@ -249,8 +252,8 @@ class RequestManager {
   /// Otherwise the returned future will be completed with a [TimeoutException].
   /// If the timeout is disabled through [disableTimeout] the future of the
   /// callback is returned immediately.
-  Future<T> timeout<T>(
-    NextcloudApiCallback<T> callback, {
+  Future<T> timeout<T>({
+    required NextcloudApiCallback<T> callback,
```

Comment:
Maybe a positional parameter is still better here.
In most cases we do not need to specify anything more.